### PR TITLE
Fix validation of nested tuples

### LINKED ISSUE #6085: Truffle-migtration-4.x/Validation of nested tuples doesn't work properly in contract's methods
In the `truffle` project, in some of their tests, they pass in a method of a contract a nested tuple as an argument.
```
let result = await deployedContract.run([[7, -5], 3], 1, address, address);
```
However, during the migration to `4.x` I noticed that the validation fails with the error:
```
    Web3 validator found 3 error[s]:
    value "7,-5" at "/0/1" must pass "uint256" validation
    value "7,-5,3" at "/1" must pass "uint8" validation
    value "1" at "/2" must pass "address" validation
   
```
which indicates that our library doesn't parse right the arguments.

This happens in two tests (however, they use the same call):

https://github.com/nazarhussain/truffle/blob/20b70910c79683cdb4b4d983e95b05476fed165c/packages/decoder/test/current/test/downgrade-test.js#L83

https://github.com/nazarhussain/truffle/blob/20b70910c79683cdb4b4d983e95b05476fed165c/packages/decoder/test/current/test/downgrade-test.js#L103

### PR DESCRIPTION
**Pull request by avkos**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../CONTRIBUTIONS.md).

Fixes #6085 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: packages/web3-validator/src/utils.ts [hunk 1]
```diff
@@ -149,11 +149,19 @@ export const abiSchemaToJsonSchema = (
...
-			// If its provided in short form of tuple e.g. [['uint', 'string']]
+			// If it's provided in short form of tuple e.g. [['uint', 'string']]
 		} else if (Array.isArray(abi)) {
 			// If its custom tuple e.g. ['tuple[2]', ['uint', 'string']]
-			if (abi[1] && Array.isArray(abi[1])) {
-				abiType = abi[0] as string;
+			if (
+				abi[0] &&
+				typeof abi[0] === 'string' &&
+				String(abi[0]).startsWith('tuple') &&
```
Comment (luu-alex) [inline packages/web3-validator/src/utils.ts]:
```suggestion
			abi[0].startsWith('tuple') &&
```
I dont think casting to string is neccesary

File: packages/web3/test/integration/external-providers/in3.test.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ import Web3 from '../../../src/index';
 // Actually it could take long time to get something from `in3` because of its decentralized nature.
 // 	And because of that, this test only try to get the last block number. And this also simplifies the configuration needed.
 describe('compatibility with `in3` provider', () => {
-	it('should get last block number', async () => {
+	it.skip('should get last block number', async () => {
```
Comment (luu-alex) [inline packages/web3/test/integration/external-providers/in3.test.ts:27]: 👍  should also create an issue for this, if we haven't already

Comment (avkos) [reply packages/web3/test/integration/external-providers/in3.test.ts:27]: we don't need. it is just limit problem. so this test will work from 1st of june

File: packages/web3-validator/src/utils.ts (hunk 1 above)
Comment (avkos) [reply packages/web3-validator/src/utils.ts]: this one need to be sure the method startsWith exists

**Review by nikoulai — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Review by jdevcs — APPROVED** [review]

**Pull request merged**